spec/lib/repositories/records/distances_spec.rb

```diff
@@ -0,0 +1,126 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe Repositories::Records, :clear_db do
+  let(:years) { nil }
+  let(:sport_types) { nil }
+
+  context 'without sessions available' do
+    it { expect(described_class.distance(years:, sport_types:)).to eq({}) }
+  end
+
+  context 'when sessions exist' do
+    before do
+      FactoryBot.create(:sport_session,
+                        duration:   3423,
```

Comment:
In the `average_pace_spec` you used duration as a `ms` value.
Here's it's seconds I think.

Reply:
Yes, will fix this, or just remove it, as it is not relevant for those tests.